ipaserver/install/ntpinstance.py

```diff
@@ -60,20 +59,7 @@ def __write_config(self):
         self.fstore.backup_file(paths.NTP_CONF)
         self.fstore.backup_file(paths.SYSCONFIG_NTPD)
 
-        # We use the OS variable to point it towards either the rhel
-        # or fedora pools. Other distros should be added in the future
-        # or we can get our own pool.
-        os = ""
-        if ipautil.file_exists(paths.ETC_FEDORA_RELEASE):
-            os = "fedora"
-        elif ipautil.file_exists(paths.ETC_REDHAT_RELEASE):
-            os = "rhel"
-
         srv_vals = []
-        srv_vals.append("0.%s.pool.ntp.org" % os)
-        srv_vals.append("1.%s.pool.ntp.org" % os)
-        srv_vals.append("2.%s.pool.ntp.org" % os)
-        srv_vals.append("3.%s.pool.ntp.org" % os)
         srv_vals.append("127.127.1.0")
         fudge = ["fudge", "127.127.1.0", "stratum", "10"]
```

Comment:
Please replace the list with single value. It will simplify the code significantly (see below).